osf/models/user.py

```diff
@@ -1261,6 +1267,13 @@ def change_password(self, raw_old_password, raw_new_password, raw_confirm_passwo
         if issues:
             raise ChangePasswordError(issues)
         self.set_password(raw_new_password)
+        self.reset_old_password_invalid_attempts()
+
+    def increment_old_password_invalid_attempts(self):
```

Comment:
Minor: This might be unnecessary indirection. I don't feel strongly about it though.

Reply:
Yes, I made it its own function to go along with the resetting invalid attempts method because I thought it read more cleanly.  If this happens..run increment function. If that happens...run reset function.  But since i'm only using once, I removed.